utils/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios"
+
+export const getArticles = (setArticles) => {
+    return axios.get(`https://nc-news-z45s.onrender.com/api/articles`).then(({data}) => {
+        setArticles(data.articles)
```

Comment:
This invoking setArticles also doesn't allow it to be reused elsewhere if you just need access to the data which comes back.